test/bigquery.test.ts

```diff
@@ -301,4 +250,677 @@ describe('BigQueryFormatter', () => {
+            ${alterTable} mydataset.mytable
+            ${alterColumn} price
+            DROP NOT NULL`;
+          const expected = dedent`
+            ${alterTable}
+              mydataset.mytable
+            ${alterColumn}
+              price
+            DROP NOT NULL`;
+          expect(format(input)).toBe(expected);
+        });
+
+        it(`Supports ${alterTable} - ${alterColumn} - SET DATA TYPE`, () => {
+          const input = `
+            ${alterTable} mydataset.mytable
+            ${alterColumn} price
+            SET DATA TYPE NUMERIC`;
+          const expected = dedent`
+            ${alterTable}
+              mydataset.mytable
+            ${alterColumn}
+              price
+            SET DATA TYPE
+              NUMERIC`;
+          expect(format(input)).toBe(expected);
+        });
+      });
+    });
+
+    const alterViews = ['ALTER VIEW', 'ALTER VIEW IF EXISTS'];
+    const alterMaterializedViews = ['ALTER MATERIALIZED VIEW', 'ALTER MATERIALIZED VIEW IF EXISTS'];
+    alterViews.concat(alterMaterializedViews).forEach(alterView => {
+      it(`Supports ${alterView} - SET OPTIONS`, () => {
+        const input = `
+          ${alterView} mydataset.myview
+          SET OPTIONS (
+            expiration_timestamp=TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL 7 DAY)
+          )`;
+        const expected = dedent`
+          ${alterView}
+            mydataset.myview
+          SET OPTIONS
+            (
+              expiration_timestamp = TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL 7 DAY)
+            )`;
+        expect(format(input)).toBe(expected);
+      });
+    });
+
+    it(`Supports ALTER BI_CAPACITY - SET OPTIONS`, () => {
+      const input = `
+        ALTER BI_CAPACITY my-project.region-us.default
+        SET OPTIONS(
+          size_gb = 250
+        )`;
+      const expected = dedent`
+        ALTER BI_CAPACITY
+          my-project.region-us.default
+        SET OPTIONS
+          (size_gb = 250)`;
+      expect(format(input)).toBe(expected);
+    });
+  });
+
+  describe('BigQuery DDL Drop Statements', () => {
+    const dropSchemas = ['DROP SCHEMA', 'DROP SCHEMA IF EXISTS'];
+    dropSchemas.forEach(dropSchema => {
+      it(`Supports ${dropSchema}`, () => {
+        const testSqls = [
+          {
+            input: `${dropSchema} mydataset`,
+            expected: dedent`
+              ${dropSchema}
+                mydataset`,
+          },
+          {
+            input: `${dropSchema} mydataset CASCADE`,
+            expected: dedent`
+              ${dropSchema}
+                mydataset CASCADE`,
+          },
+          {
+            input: `${dropSchema} mydataset RESTRICT`,
+            expected: dedent`
+              ${dropSchema}
+                mydataset RESTRICT`,
+          },
+        ];
+
+        testSqls.forEach(testSql => expect(format(testSql.input)).toBe(testSql.expected));
+      });
+    });
+
+    const dropTables = ['DROP TABLE', 'DROP TABLE IF EXISTS'];
```

Comment:
these lists can be declared as a map object, ie.
```js
const dropKeywordTypes = {
  dropTables: [...],
  // etc.
}
```

and you can collect them with the `flatKeywordLists` util function